nova/virt/lxd/vif.py

```diff
@@ -23,7 +23,7 @@
 from nova.network import os_vif_util
 
 import os_vif
-
+from vif_plug_ovs import linux_net as linux_net_ovs
```

Comment:
This file contains is the patch that is currently applied via Ansible. Once this custom module is published to Nexus, I'll be able to change the Ansible roles to not apply that patch and `pip install` this custom module.